### PR TITLE
Get rid of ActiveXML for unlocking projects

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: src/api/app/controllers/webui/project_controller.rb [hunk 1]
```diff
@@ -875,14 +875,12 @@ def unlock_dialog
   end
 
   def unlock
-    begin
-      path = "/source/#{CGI.escape(params[:project])}/?cmd=unlock&comment=#{CGI.escape(params[:comment])}"
-      frontend.transport.direct_http(URI(path), :method => 'POST', :data => '')
```
Comment (coolo) [inline src/api/app/controllers/webui/project_controller.rb]: The comment was never used?

Comment (adrianschroeter) [inline src/api/app/controllers/webui/project_controller.rb]: On Dienstag, 20. Oktober 2015, 04:23:33 CEST wrote Stephan Kulow:
> > @@ -875,14 +875,12 @@ def unlock_dialog
> >    end
> >  
> >    def unlock
> > -    begin
> > -      path = "/source/#{CGI.escape(params[:project])}/?cmd=unlock&comment=#{CGI.escape(params[:comment])}"
> > -      frontend.transport.direct_http(URI(path), :method => 'POST', :data => '')
> 
> The comment was never used?

an unlock without a comment will fail.

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany

Comment (hennevogel) [inline src/api/app/controllers/webui/project_controller.rb]: The whole webui function was never used because the link was hidden...

Comment (hennevogel) [inline src/api/app/controllers/webui/project_controller.rb]: I've added the comment to the webui method...

**Comment by hennevogel** [comment]
Can anyone please review? :-) @openSUSE/open-build-service

**Pull request merged**